Woof/Woof/Helper/AppStyle/UIElementSize.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+extension AppStyle {
+    /// A utility enum defining various sizes for UI elements.
+    enum UIElementSize {
+        /// The stroke size used for borders.
+        static let stroke = 12.0
+
+        /// The corner radius size used for rounded corners.
```

Comment:
nit:
```suggestion
        /// The default corner radius size used for rounded corners.
```
?

Review the rest of code.

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/42/commits/f6ce719f1bf56b406708b8bce981f36ca98173c0